blog/_posts/2016-11-29-intercontinental-gotober.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Intercontinental conference meeting @GOTOber in Berlin
+layout: post
+created_at: Tue Nov 29 2016
+permalink: blog/2016-11-29-intercontinental-gotober
+current: blog
+author: Marie & Thea, Taneea & Vrinda, Mayar & Nada
+twitter: LoadToCode
+categories:
+- student-posts
+- conferences
+---
+![RGSoC goes GOTOber](/img/blog/2016/20161128-GOTOber_1.gif)<font color="grey"><small><i>Team Twitches, Team Ruby's Secret and Team LoadToCode go GOTOber</i></small></font>
```

Comment:
This image (and all other ones in the post) need an additional image credit: either the name of the person who took the photos or the name of the team member the photo "belongs to". (i.e. if you asked a stranger to take a photo, just put down the name of the team member whose phone was used for the photo).